src/timer/timer_game_economy.cpp

```diff
@@ -38,6 +38,41 @@ TimerGameEconomy::Month TimerGameEconomy::month = {};
 TimerGameEconomy::Date TimerGameEconomy::date = {};
 TimerGameEconomy::DateFract TimerGameEconomy::date_fract = {};
 
+/**
+ * Converts a Date to a Year, Month & Day.
+ * @param date the date to convert from
+ * @param ymd  the year, month and day to write to
+ */
+/* static */ void TimerGameEconomy::ConvertDateToYMD(TimerGameEconomy::Date date, TimerGameEconomy::YearMonthDay *ymd)
+{
+	/* If we're using real-time units, economy months have 30 days and an economy year has 360 days.
+	 * Otherwise, we need to keep economy months and years in sync with the calendar. */
+	if (::_settings_game.economy.use_realtime_units) {
+		ymd->year = TimerGameEconomy::date.base() / EconomyTime::DAYS_IN_ECONOMY_YEAR;
+		ymd->month = TimerGameEconomy::date.base() % EconomyTime::DAYS_IN_ECONOMY_YEAR / EconomyTime::DAYS_IN_ECONOMY_MONTH;
```

Comment:
please add `()`. Although the order is well defined, it just makes reading it a bit harder. So:

`(a % b) / c` please.